src/constans/TypeproductImg.js

```diff
@@ -3,12 +3,12 @@ export const TYPEIMG = {
     SRC: "https://static.wixstatic.com/media/c77c68_f7328ff35fa04b51b29e3a7edc785734~mv2.png/v1/crop/x_343,y_0,w_1362,h_2048/fill/w_213,h_320,al_c,q_85,usm_0.66_1.00_0.01,enc_auto/IMG-1226_edited.png",
     ALT: "ring",
     BUTTON: "Rings",
-     },
-     BRACELETS: {
-          SRC: "https://static.wixstatic.com/media/c77c68_5ec1285841c843ef8f6c68ae1bbb9bfd~mv2.png/v1/crop/x_171,y_0,w_1707,h_2048/fill/w_234,h_281,al_c,q_85,usm_0.66_1.00_0.01,enc_auto/IMG-1228_edited.png",
-          ALT: "bracelets",
-          BUTTON: "Bracelets",
-        },
+  },
+  BRACELETS: {
+    SRC: "https://static.wixstatic.com/media/c77c68_5ec1285841c843ef8f6c68ae1bbb9bfd~mv2.png/v1/crop/x_171,y_0,w_1707,h_2048/fill/w_234,h_281,al_c,q_85,usm_0.66_1.00_0.01,enc_auto/IMG-1228_edited.png",
+    ALT: "bracelets",
```

Comment:
If you want to load images from the internet, maybe save this string into variable/file in order to make you code cleaner